docs/route.md

```diff
@@ -907,27 +927,3 @@ Vue.component("my-child", {
 	},
 })
 ```
-
-Technically, there's nothing stopping you from even doing it in a Mithril component, even.
-
-```javascript
-// Don't do this. Use a proper global layout component for each route instead,
-// passing your child vnode/component in the attributes or children.
-function Child() {
-	return {
-		oncreate: function(vnode) {
-			m.route(vnode.dom, "/", {
-				// ...
-			})
-		},
-
-		onremove: function() {
-			m.mount(vnode.dom, null)
-		},
-
-		view: function() {
-			return m("div")
-		},
-	}
-}
-```
```

Comment:
Why are you removing this?

Reply:
It’s redundant / impertinent — a funny observation in conversation but unhelpful in the context of documenting functionality some readers may find difficult to follow.